### ISSUE TITLE
Cannot bundle top-level $ref

### ISSUE THREAD

**Issue by morris** [BigstickCarpet/swagger-cli#12]
Hi,

it looks like swagger-cli cannot bundle a top-level $ref even if it points to a valid Swagger file:

```
$ swagger validate http://petstore.swagger.io/v2/swagger.json
http://petstore.swagger.io/v2/swagger.json is valid
$ cat test.yaml
$ref: http://petstore.swagger.io/v2/swagger.json
$ swagger bundle test.yaml
SyntaxError: test.yaml is not a valid Swagger API definition
    at Function.ono [as syntax] (...\npm\node_modules\swagger-cli\node_modules\ono\lib\index.js:61:20)
    at ...\npm\node_modules\swagger-cli\node_modules\swagger-parser\lib\index.js:75:19
```

Thanks and keep up the good work :+1:

**Comment by morris** [comment]
P.S. this is of course a swagger-parser issue, should've reported it there - sorry. It also applies to json-schema-ref-parser/bundle function, which just returns the $ref untouched.

**Comment by BigstickCarpet** [comment]
@morris - Thanks for reporting this.  It may actually be a Swagger CLI issue, since Swagger CLI uses an outdated version of Swagger Parser.  Either way... I'll look into it and get it fixed.

**Comment by BigstickCarpet** [comment]
I just released Swagger-CLI v1.0.0, which includes the latest version of Swagger-Parser.  So this issue should now be fixed.
